mmdet/datasets/coco.py

```diff
@@ -525,7 +525,7 @@ def evaluate_det_segm(self,
 
                 for item in metric_items:
                     val = float(
-                        f'{cocoEval.stats[coco_metric_names[item]]:.3f}')
+                        f'{cocoEval.stats[coco_metric_names[item]]:.4f}')
```

Comment:
Since you changed the float precision of the metric, the unit test should also be updated.